<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1">
      <item>
       <widget class="QTableView" name="_bayesTableView"/>
      </item>
      <item>
       <widget class="QTableView" name="_resultsView"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="_classifierInput"/>
      </item>
      <item>
       <widget class="QPushButton" name="_computeButton">
        <property name="text">
         <string>Compute</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="_classificationLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="_actionOpenTrainingDir"/>
    <addaction name="_actionOpenFile"/>
    <addaction name="_actionSave"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="_actionOpenTrainingDir">
   <property name="text">
    <string>Open Training Folder</string>
   </property>
  </action>
  <action name="_actionSave">
   <property name="text">
    <string>Save Classifier</string>
   </property>
  </action>
  <action name="_actionOpenFile">
   <property name="text">
    <string>Open Classifier</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
